### PR TITLE
Add LUXMICROAIO

### PR DESCRIPTION
**Pull request by freasy**

### CODE REVIEW

File: configs/LUXMICROAIO/config.h [hunk 1]
```diff
@@ -0,0 +1,123 @@
...
+#define USE_FLASH_W25Q128FV
+#define USE_GYRO_SPI_ICM42688P
+#define USE_SPI_GYRO
+
+#define USE_BARO_DPS310
+#define USE_BARO_BPM280
+#define USE_BARO_BPM085
+#define USE_BARO_MS5611
+#define USE_BARO_QMP6988
+#define USE_BARO_LPS
```
Comment (haslinghuis) [inline configs/LUXMICROAIO/config.h:45]: Please only specify physical present barometer drivers. Two drivers should be the max allowed.

Comment (nerdCopter) [reply configs/LUXMICROAIO/config.h:45]: @freasy

Comment (ot0tot) [reply configs/LUXMICROAIO/config.h:45]: You can also only define USE_BARO which includes all of the baro drivers by default.

Comment (haslinghuis) [reply configs/LUXMICROAIO/config.h:45]: No this hack is not a recommended practice

**Comment by mr-sour** [comment]
I have one of these boards as they're for sale now. they have DPS310 could we go forward with that so these can be updated.
image

**Review by haslinghuis — APPROVED** [review]

File: configs/LUXMICROAIO/config.h (hunk 1 above)
Comment (haslinghuis) [reply configs/LUXMICROAIO/config.h]: Updated the config

**Comment by mr-sour** [comment]
image
<3

**Pull request merged**